app/views/users/sessions/new.html.erb

```diff
@@ -1,28 +1,43 @@
 <% if params[:restricted] %>
-  <h3 class="heading"><%= ts('Sorry!') %></h3>
+  <h3 class="heading"><%= ts("Sorry!") %></h3>
   <p>
-    <%= ts('This work is only available to registered users of the Archive.') %>
-    <%= ts('If you already have an Archive of Our Own account, log in now.') %>
-    <% if @admin_settings.account_creation_enabled? %>
-      <%= link_to ts("Or join us for free - it's easy."), signup_path %>
+    <%= ts("This work is only available to registered users of the Archive.") %>
+    <%= ts("If you already have an Archive of Our Own account, log in now.") %>
+      <% if @admin_settings.invite_from_queue_enabled? && @admin_settings.creation_requires_invite? %>
+        <%= ts("If you don't have an account, you can") %> <%= link_to ts("request an invitation"), invite_requests_path %> <%= ts("to join.") %>
+      <% elsif @admin_settings.account_creation_enabled? && !@admin_settings.creation_requires_invite? %>
+        <%= link_to ts("Or join us for free - it's easy."), new_user_path %>
     <% end %>
   </p>
+<% elsif params[:high_load] %>
```

Comment:
Something went wrong in the merge, we removed this block in #3388.

Reply:
👍?